src/linker/Linker/LinkContext.cs

```diff
@@ -292,8 +292,14 @@ static AssemblyNameReference GetReference (IMetadataScope scope)
 
 			return reference;
 		}
+		
+		public void SetAction(AssemblyDefinition assembly, AssemblyAction action)
+		{
+			RegisterAssembly(assembly);
+			Annotations.SetAction(assembly, action);
+		}
 
-		protected void SetAction (AssemblyDefinition assembly)
+		protected void SetActionToDefault (AssemblyDefinition assembly)
```

Comment:
Shouldn't that be `SetDefaultAction` ?

Reply:
Fixed.  I was on the fence and almost called it that originally.